dev/bots/suite_runners/run_tool_tests.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:path/path.dart' as path;
+
+import '../utils.dart';
+
+Future<void> toolTestsRunner() async {
```

Comment:
Was there a particular reason the tool integration tests are not here?

Reply:
I can create a suite for all those tool tests to be included if that would make more sense

Reply:
I'm not sure what that means. How much work does "creating a suite" entail? Although I suppose this was in its own function in test.dart. I suspect that was accidental, rather than intentional.

From a purely organizational perspective, I would weakly prefer that all the test functions that invoke test directories under `//flutter/packages/flutter_tools/test` be grouped together in the same library (dart's terminology for a file). But if that's a lot of work to implement I'm open to discussion.